src/Assets/Scripts/GameController.cs

```diff
@@ -18,6 +24,7 @@ public class GameController : MonoBehaviour
     private Vector2 direction = Vector2.down;
     [SerializeField]
     public float speed = 1;
+    public float maxSpeed = 15;
```

Comment:
```suggestion
    private float speed = 1;
    private float maxSpeed = 15;
```
kan göra båda till private